### PR TITLE
ESSI-121 Adds OCR indicators in file manger and download link on file details.

### PR DESCRIPTION
**Pull request by andjsmit**

### CODE REVIEW

**Comment by rwingiupui** [comment]
👍 +1

File: config/initializers/add_extracted_text_downloads.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Rails.application.config.after_initialize do
+  Hyrax::DownloadsController.class_eval do
+    prepend DownloadsControllerExtensions
+  end
+end
+
+# Makes extracted text check available in file details.
+module FileSetPresenterExtensions
+  def extracted_text?
+    return true if ::FileSet.find(id).extracted_text
```
Comment (aploshay) [inline config/initializers/add_extracted_text_downloads.rb]: could condense to 1 line with: `::FileSet.find(id).extracted_text.present?` ?

File: app/views/hyrax/file_sets/_show_details.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<dl class="dl-horizontal file-show-term file-show-details">
+  <dt>Depositor</dt>
+  <dd itemprop="accountablePerson" itemscope itemtype="http://schema.org/Person"><span itemprop="name"><%= link_to_profile @presenter.depositor %></span></dd>
+  <dt>Date Uploaded</dt>
+  <dd itemprop="datePublished"><%= @presenter.date_uploaded %></dd>
+  <dt>Date Modified</dt>
+  <dd itemprop="dateModified"><%= @presenter.date_modified %></dd>
+  <% if @presenter.extracted_text? %>
+  <dt>OCR</dt>
+  <dd><%= link_to 'Download', "/downloads/#{@presenter.id}?file=extracted_text" %></dd>
```
Comment (aploshay) [inline app/views/hyrax/file_sets/_show_details.html.erb]: is there a more orthodox way to build this URL?  That's a genuine question; I don't object to what's here currently

File: config/initializers/add_extracted_text_downloads.rb (hunk 1 above)
Comment (andjsmit) [reply config/initializers/add_extracted_text_downloads.rb]: Yes - reads better - thanks.

File: app/views/hyrax/file_sets/_show_details.html.erb (hunk 2 above)
Comment (andjsmit) [reply app/views/hyrax/file_sets/_show_details.html.erb]: Not sure of a better way to form URL, but I moved the link creation to the presenter so all extracted_text stuff resides in same file.

**Pull request merged**